keyboards/sirius/unigo66/unigo66.h

```diff
@@ -120,4 +120,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     KE1,KC_NO,K1D,K1B,K06,K19,K05,K11,K10,K36,K37,KC_NO,    KC_NO,KE5,      K52,      KC_NO,KC_NO,KC_NO,KC_NO, KC_NO,KC_NO, \
     KE0,KE3,KE2,KC_NO,KC_NO,    K2C,    KC_NO,KC_NO,KC_NO,KE6,KC_NO,KC_NO,KE4,  K50,K51,K4F,  KC_NO,      KC_NO,KC_NO, KC_NO,KC_NO  \
 )
-#endif
+
+#define LAYOUT_BetaPCB( \
```

Comment:
I don't know the naming for it, but usually, everything after `LAYOUT` is lower case.  Specifically:

```suggestion
#define LAYOUT_beta_pcb( \
```

Reply:
fixed in both unigo66.h and info.json